doc/nrf/releases_and_maturity/migration/migration_3.1_54h_suit_ironside.rst

```diff
@@ -0,0 +1,116 @@
+* You have a working |NCS| 3.0 application for the nRF54H20 SoC using SUIT.
+* You have installed |NCS| 3.1 and its toolchain.
+
+.. note::
+   To program IronSide SE on your nRF54H20 SoC-based device, your device must be in lifecycle state (LCS) ``EMPTY``.
+   Devices using SUIT in LCS RoT cannot be transitioned back to LCS EMPTY.
+
+Breaking Changes
+================
+
+The following is a summary of the breaking changes that apply when migrating applications:
+
+* SUIT support is removed in |NCS| 3.1.
+  The Secure Domain now runs IronSide SE, which uses MCUboot for device firmware update.
+* IronSide SE uses a new UICR format and moves the peripheral configuration into a dedicated MRAM partition.
+* Shared memory (IPC) is relocated to a fixed RAM-20 region.
+  You no longer reserve it manually.
+* DFU support moves from SUIT to MCUboot.
+  You must enable and configure MCUboot in your project.
+
+Update prj.conf
+---------------
+
+Remove the following SUIT-specific Kconfig options:
+
+* :kconfig:option:`CONFIG_SUIT`
+* :kconfig:option:`CONFIG_SUIT_SECURE_DOMAIN`
+
+Enable MCUboot for device firmware update (DFU) in the :file:`sysbuild.conf` file as follows:
+
+* Set :kconfig:option:`SB_CONFIG_BOOTLOADER_MCUBOOT` to ``y``.
+
+
+Update devicetree files
+-----------------------
+
+You must update your devicetree files as follows:
+
+* Remove the old UICR partition.
+  In your board's DTS overlay, remove any node that defined the ``uicr`` partition.
+
+* Add the IronSide ``peripconf`` partition.
+  IronSide expects a blob of address-value pairs called ``peripconf`` stored in the MRAM.
+
+* Remove IPC-shared-memory reservation.
+  As IronSide relocates the IPC buffer to a fixed RAM20 address, you can delete any manual reservation in RAM0.
+  Refer to the `Memory map changes`_ section.
+
+Update peripheral configuration (peripconf)
+-------------------------------------------
+
+The new UICR format no longer holds peripheral initial values.
+You must generate a peripconf blob at build time.
+
+* Zephyr build applies an ``nrf_rpc_peripconf`` script.
+* The script reads ``peripconf`` nodes in the DTS.
+* It outputs a binary blob and embeds it into the MRAM partition.
+
+You do not need to modify your application code.
+You only need to ensure the DTS partition exists.
+
+Memory protection and isolation
+-------------------------------
+
+IronSide SE currently grants full memory-access permissions to both application and radio domains by default.
+Delete any UICR settings related to the following:
+
+  * IPC location
+  * Secure-Domain offsets
+  * Partition lock bits
+
+Memory map changes
+------------------
+
+With IronSide SE, the memory map changed as follows:
+
+* The application core firmware now always starts at address ``0xE03_0000``, which is the first address in ``MRAM00`` immediately following the IronSide firmware.
+* The radio core is not started automatically.
+* Nordic-reserved partitions in ``MRAM10`` and ``RAM0x`` have been removed.
```

Comment:
MRAM11* - we still reserve the start of MRAM10 (everything before 0xE03_0000) for the IronSide firmware